docker/README.md

```diff
@@ -0,0 +1,64 @@
+# docker
+
+  * `acra-build.dockerfile` - intermediate image for compile all acra components
+  * `acraserver.dockerfile` - resulting image with acraserver
+  * `acraproxy.dockerfile` - resulting image with acraproxy
+  * `acra_genkeys.dockerfile` - resulting image with acra_genkeys
+
+## Build containers
+
+```
+make docker
+```
+
+# docker-compose
+
+  * `docker-compose-pgsql-ssession.yml` - postgresql, acraserver, acraproxy
+    with SecureSession interconnection
+
+
+## Quick launch
+
+!!!!!!!!!!!!!!!!!!!!!!!!!!!
+!!! INSECURE, TEST ONLY !!!
+!!!!!!!!!!!!!!!!!!!!!!!!!!!
+```
+docker-compose -f docker/<compose_file_name>.yml up
+```
+This will create `docker/.acrakeys` directory structure, generate all key pairs,
+put them to appropriate servcies' directories and launch all components.
+
+Now you can connect to 9494/tcp (acraproxy) or 5432/tcp (postgresql).
+
+
+## Normal launch
+
+Please specify ACRA_MASTER_KEY:
+```
+export ACRA_MASTER_KEY=$(echo -n "My_Very_Long_Key_Phrase_ge_32_chars" | base64)
```

Comment:
if `ge` == great equal, then we need to update our master key check and change `==` sign to `>=`. I think I will do it in a future

Reply:
OK, so I keep this wording.